middleware/lib/sfMiddleware.ts

```diff
@@ -188,25 +183,24 @@ export function applySfFnMiddleware(request: any, logger: Logger): Array<any> {
         //change the logger level, so any subsequent user function's logger.debug would log as well
         logger.setLevel(LoggerLevel.DEBUG);
         logger.debug('debug raw request in middleware');
-        logger.debug(request);
+        logger.debug(`headers=${JSON.stringify(headers)}`);
+        logger.debug(cloudEvent);
```

Comment:
Wonder if this'll log sensitive info or GDPR PPI.

Reply:
Good call - I'll just log the keys.

Reply:
Email or UserId are others.  Let's leave for now.